src/components/MenuBubble.vue

```diff
@@ -113,11 +114,21 @@ export default {
 	},
 	data: () => {
 		return {
+			isActive: false,
 			linkUrl: null,
 			linkMenuIsActive: false,
 			isUsingDirectEditing: loadState('text', 'directEditingToken', null) !== null,
 		}
 	},
+	mounted() {
+		this.$_updateIsActive = debounce(this.updateIsActive.bind(this), 50)
+		this.$editor.on('update', this.$_updateIsActive)
```

Comment:
I'm wondering if we actually need this.
Any change to the content in question (i.e. someone else turning the selection into a link) should also result in changes to the absolute position of the selection in the text because new tags are inserted. Not sure if that triggers a `selectionUpdate` though. Will need to try it out.

Reply:
Okay... so i just checked and we need to act on `update` as well. Even when we change the link ourselves the menububble would not update otherwise.